rest/data/project_test.go

```diff
@@ -43,6 +87,30 @@ func TestProjectConnectorGetSuite(t *testing.T) {
 			}
 		}
 
+		before := sampleProjectSettings
+		after := before
```

Comment:
I would have a function that returns your mock structure. also this double assign+declare as a way to do a copy isn't great.